gen/build_deploy/bash/dcos_generate_config.sh.in

```diff
@@ -50,4 +53,21 @@ else
     trap 'docker kill {genconf_tar}' HUP QUIT INT TERM
     docker run --rm --name=$DCOS_INSTALLER_CONTAINER_NAME -i -p $PORT:9000 -v $(pwd)/genconf/:/genconf {docker_image_name} "$@"
 fi
-exit $?
+
+if [ $? -ne 0 ]; then
+    echo "Problems running installer"
```

Comment:
Could we make this message more useful?

Reply:
I'll remove this message completely. The installer itself should emit a meaningful error message.